cups/ppd.c

```diff
@@ -3075,7 +3075,7 @@ ppd_read(cups_file_t    *fp,		/* I - File to read from */
 	  colon = 1;
 
 	if (ch == '\"' && colon)
-	  endquote = !endquote;
+	  endquote ^= 1;
```

Comment:
I think the original code better expresses the intent.

Reply:
C, unlike C++, is less efficient than C++ in this one area, where bool negation is implemented by most compilers as XOR 1, but because endquote is a whole integer, it has to be turned into a conditional in itself.

Hence this change

Reply:
I get that, but this isn't performance-critical code and doing XOR for boolean negation is not as clear semantically as the unary negation operator.